setup.py

```diff
@@ -60,8 +60,10 @@
     ],
     "gpu": [
         "nvidia-ml-py3>=7.352.0",
-        "tensorflow-gpu>=1.15.0,<2",  # compiled with CUDA 10.0
-        "torch==1.2.0",  # last os-common version with CUDA 10.0 support
+        "tensorflow-gpu==2.3.0",  # compiled with CUDA 10.0
+        "torch==1.9.1",  # last os-common version with CUDA 10.0 support
+        "six~=1.15.0",
+        "typing-extensions~=3.7.4",
```

Comment:
same thing here, at this point if we add TF 2 we will break the repo
@anargyri is working in the transition, but so far we can't add this change